src/components/events/EventBox.tsx

```diff
@@ -20,19 +20,43 @@ const T1BG = styled('div')`
   background-position: center center;
   transition: filter 0.5s ease;  
 `
+
+interface eventPreviewData {
+  allEventsJson: {
+      nodes: {
+        Year: string,
+        Information: {
+          Title: string
+        }[]
+      }[]
+  }
+}
 export const EventsBox: React.FC<{}> = () => {
+        const data = useStaticQuery(
+       graphql`
+        query {
+          allEventsJson {
+            nodes {
+              Year
+              Information {
+                Title
+              }
+            }
+          }
+      }`
+     )
+     const info = data.allEventsJson.nodes;
+     const allYears = info.map((year) => {
 
-    const allYears = totalevents.map((year) => {
-      
       const eventsThisYear = year.Information.map((event) => {
         return(
-        <a className="text-container" href={"/years/".concat(year.Year).concat('/#').concat(event.Title)}>
+        <a className="text-container" key={event.Title} href={"/events/".concat(year.Year).concat('/#').concat(event.Title)}>
```

Comment:
I think using string template literals `/events/${year.Year}` etc would be cleaner than chaining concats here

Reply:
i agree

Reply:
Hm I'm not sure why I didn't do this before. I think for some reason the template didn't work but concats did. But can't reproduce so maybe I was just ded lol